src/main/resources/assets/securitycraft/lang/fr_fr.json

```diff
@@ -0,0 +1,767 @@
+	"gui.securitycraft:scManual.recipe.adminTool": "L’outil d’administration est uniquement disponible via l’inventaire, s’il est activé via le fichier de configuration.",
+	"gui.securitycraft:scManual.recipe.admin_tool": "L’outil d’administration est uniquement disponible via l’inventaire, s’il est activé via le fichier de configuration..",
+	"gui.securitycraft:scManual.recipe.reinforced_crystal_quartz": "Fabriqué à partir de Bloc de Cristal de Quartz ordinaire dans un Blindeur de blocs universel.",
+	"gui.securitycraft:scManual.recipe.sc_manual": "Le manuel SecurityCraft est uniquement disponible via l’inventaire ou en tapant \"/sc help\" (sans apostrophes).",
+	"gui.securitycraft:scManual.reinforced": "Blocs blindés",
+	"gui.securitycraft:scManual.designedBy": "Conçu par: %s",
+	"gui.securitycraft:ims.hostileAndPlayers": "Mobs hostiles & joueurs",
+	"gui.securitycraft:ims.hostile": "Mobs hostiles",
+	"gui.securitycraft:ims.target": "Cible:",
+	"gui.securitycraft:invScan.emitredstone": "Emettre redstone",
+	"gui.securitycraft:invScan.checkInv": "Vérifier l’inventaire",
+	"gui.securitycraft:invScan.explanation.1": "Ce paramètre vérifiera l’inventaire",
+	"gui.securitycraft:invScan.explanation.2": "d’un joueur, et, s’il contient un",
+	"gui.securitycraft:invScan.explanation.checkInv.3": "élément interdit, l’effaçera",
+	"gui.securitycraft:invScan.explanation.checkInv.4": "de l’inventaire.",
+	"gui.securitycraft:invScan.explanation.emitredstone.3": "élément interdit, émission d’un signal ",
+	"gui.securitycraft:invScan.explanation.emitredstone.4": "redstone pendant 3 secondes.",
+	"gui.securitycraft:invScan.setTo": "Ce scanner est réglé sur :",
+	"gui.securitycraft:invScan.prohibitedItems": "Eléments interdits",
+	"gui.securitycraft:invScan.mode.admin": "Mode Admin",
+	"gui.securitycraft:invScan.mode.view": "Mode consultation",
+	"gui.securitycraft:invScan.storage": "Stockage",
+	"gui.securitycraft:keycardSetup.equal": "égal à",
+	"gui.securitycraft:keycardSetup.equalOrHigher": "supérieur ou égal à",
+	"gui.securitycraft:keycardSetup.save": "enregistrer & continuer",
+	"gui.securitycraft:keycardSetup.explanation.1": "Installation de Carte d‘accès",
+	"gui.securitycraft:keycardSetup.explanation.2": "Sélectionner le niveau de sécurité",
+	"gui.securitycraft:keycardSetup.explanation.3": "voulu. Seuls les joueurs avec une",
+	"gui.securitycraft:keycardSetup.explanation.4": "carte d’accès ayant le bon ",
+	"gui.securitycraft:keycardSetup.explanation.5": "niveau de sécurité pourront ",
+	"gui.securitycraft:keycardSetup.explanation.6": "utiliser ce lecteur de",
+	"gui.securitycraft:keycardSetup.explanation.7": "cartes.",
+	"gui.securitycraft:keycardSetup.lvlNeeded": "Niveau de sécurité requis:",
+	"gui.securitycraft:protectedFour.name": "Four Protégé",
```

Comment:
You accidentally changed the language key again. `protectedFour` should be `protectedFurnace`

Reply:
Still needs to be changed.